instrumentation/gwt-2.0/javaagent/gwt-2.0-javaagent.gradle

```diff
@@ -59,11 +54,10 @@ task compileGwt(dependsOn: classes, type: JavaExec) {
   def extraDir = "$buildDir/testapp/extra"
   def warDir = "$buildDir/testapp/war"
 
-  outputs.dir warDir
+  outputs.cacheIf { true }
```

Comment:
Can you explain this?

Reply:
Yeah it's from this https://docs.gradle.org/current/userguide/build_cache.html#using_the_runtime_api